index.html

```diff
@@ -13,29 +15,29 @@
 
     <h1 id="title">Our Demo Survey form</h1>
     <p id="description">Thank you for taking your time reviewing and filling this form</p>
-    <form action="" id="survey-form">
-      <div class="required">
+    <form id="survey-form">
+      <DIV class="required">
         <label for="name" id="name-label">Name</label><input type="text" name="name" id="name" placeholder="Enter your name" required>
         <label for="email" id="email-label">Email</label><input type="email" name="email" id="email" placeholder="Enter your email" required>
         <label for="number" id="number-label">Number</label><input type="number" name="number" id="number" min="5" max="20" placeholder="Enter your number">
-      </div>
+      </DIV>
       
       <select name="dropdown" id="dropdown">
         <option value="Front-End">Front End</option>
         <option value="Back-End">Back End</option>
       </select>
       
-      <div class="radio">
+      <div class = radio>
```

Comment:
- [ ] Please always quote attribute values and use space-less equal signs.

Reply:
- [ ] You did a great review overall :+1: . I think it is always nice if you include an explanation, a "why", for each requested change. This helps students get a lesson from every change they are requested to make. :pray:

Reply:
- [ ] Please always quote attribute values and use space-less equal signs because you are required to follow the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
- [ ] OPTIONAL: As it is great to direct the student to links for reference you could have your brief explanation of the 'why' with a few words. :pray: